code/__DEFINES/species.dm

```diff
@@ -1,12 +1,12 @@
 // Pressure limits.
 /// This determins at what pressure the ultra-high pressure red icon is displayed. (This one is set as a constant)
-#define HAZARD_HIGH_PRESSURE 550
+#define HAZARD_HIGH_PRESSURE 303
 /// This determins when the orange pressure icon is displayed (it is 0.7 * HAZARD_HIGH_PRESSURE)
-#define WARNING_HIGH_PRESSURE 325
+#define WARNING_HIGH_PRESSURE 202
 /// This is when the gray low pressure icon is displayed. (it is 2.5 * HAZARD_LOW_PRESSURE)
-#define WARNING_LOW_PRESSURE 50
+#define WARNING_LOW_PRESSURE 60
 /// This is when the black ultra-low pressure icon is displayed. (This one is set as a constant)
-#define HAZARD_LOW_PRESSURE 20
+#define HAZARD_LOW_PRESSURE 40
```

Comment:
probably update the comments here because they will be inaccurate with the current numbers

Reply:
Yep